RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -891,7 +891,7 @@ boolean ocrs_postHelper();									//Defined in autoscend/auto_summerfun.ash
 void oldPeoplePlantStuff();									//Defined in autoscend/auto_floristfriar.ash
 boolean organsFull();										//Defined in autoscend/auto_util.ash
 boolean ovenHandle();										//Defined in autoscend/auto_util.ash
-boolean pete_buySkills();									//Defined in autoscend/auto_sneakypete.ash
+void pete_buySkills();										//Defined in autoscend/auto_sneakypete.ash
```

Comment:
why change the return type here? the change seems unnecessary

Reply:
Because that function is not a boolean.

1. We never actually used it as one, not caring what it returns and in fact having had specific code to always make it return false.
We called `if(LM_pete()) return true;` whose job was todo
1. check if pete
2. run pete_buySkills()
3. return false always
in that order.
Then pete_buySkills() itself does
1. return false if not pete
2. check if there are any skills you can buy and buy them
3. return true, regardless of whether there were any skills bought. In effect it returns true if you are in a pete run and false if not.

This is just a void function that was written as a bool for some reason. And relies on a wrapper to convert it to always return false.

Reply:
> We called if(LM_pete()) return true; whose job was todo

?

I don't really agree but let's loop in @Malibu-Stacey on this, it's quite possible I'm being unreasonable. My intuition is that, even if it's currently not used, it's helpful for functions that can make global state changes to return `boolean`, just in case we want to use that functionality later. It's okay to have a function that returns `boolean` that we use in one place and immediately discard the return value of.

Reply:
To clarify and get to the crux of the issue (since I went on and rambled a bit in that post)
It is a boolean function that does not return meaningful (to what it does) values.
Even though it is a function for buying skills, it returns true if in path, false if not in path. Actual skill buying has nothing to do with its return value.